components/Modal/Modal.js

```diff
@@ -44,11 +44,12 @@ function Modal({
           className={ModalStyles.overlay}
           onClick={shouldCloseOnOverlayClick ? onRequestClose : undefined}
           data-testid={MODAL_OVERLAY}
-        />
+        >
+          <CloseButton theme="secondary" onClick={onRequestClose} />
```

Comment:
Can we use `https://www.radix-ui.com/docs/primitives/components/dialog#close` with `asChild` prop here?

Reply:
I was looking at that earlier - I wasn't sure if we should use it since there's already a CloseButton shared component, but I _think_ it still works if we just add the Dialog Close as a wrapper around it?

Reply:
Update - in the example on the Dialog spec, it does look like they're just wrapping existing buttons with the Dialog Close tag, so I can add that quickly.

Reply:
Yep! I wouldn't do it around the definition of CloseButton because the CloseButton could be used in other non-modal contexts. So, just right here is good 😁